[![DOI](https://zenodo.org/badge/99698135.svg)](https://zenodo.org/record/3349688)  

# awesome-microbiome-stats
A curated list of papers and articles on microbiome statistical analysis and tools. Constantly in development at [GitHub](https://github.com/vinisalazar/awesome-microbiome-stats/).

Microbiome studies have become extremely important and popular in recent years, and many challenges and debates have arisen regarding statistical analysis. This repository attempts to cover the most relevant papers in the topics of microbiome statistical methods, including state-of-the-art tools.

#### 2019
- [Carr, A., Diener, C., Baliga, N. S. & Gibbons, S. M. Use and abuse of correlation analyses in microbial ecology. ISME J. 13, 2647–2655 (2019).](https://www.nature.com/articles/s41396-019-0459-z)  
- [Kumar, M., Ji, B., Zengler, K. & Nielsen, J. Modelling approaches for studying the microbiome. Nat. Microbiol. 4, 1253–1267 (2019).](https://www.nature.com/articles/s41564-019-0491-9)
- [McLaren, M. R., Willis, A. D. & Callahan, B. J. Consistent and correctable bias in metagenomic sequencing experiments. Elife 8, e46923 (2019).](https://elifesciences.org/articles/46923)

#### 2018
- [Knight, R. et al. Best practices for analysing microbiomes. Nat. Rev. Microbiol. 16, 410–422 (2018).](https://www.nature.com/articles/s41579-018-0029-9)
- [McMurdie, P. J. Normalization of Microbiome Profiling Data. Methods Mol. Biol. 1849, 143–168 (2018).](https://link.springer.com/protocol/10.1007%2F978-1-4939-8728-3_10)

#### 2017
- [Callahan, B. J., Mcmurdie, P. J. & Holmes, S. P. Exact sequence variants should replace operational taxonomic units in marker-gene data analysis. ISME J. 11, 2639–2643 (2017).](https://www.nature.com/articles/ismej2017119)
- [Vandeputte, D. et al. Quantitative microbiome profiling links gut community variation to microbial load. Nature 551, 507 (2017).](https://www.nature.com/articles/nature24460)
- [Gloor, G. B., Macklaim, J. M., Pawlowsky-Glahn, V. & Egozcue, J. J. Microbiome datasets are compositional: and this is not optional. Front. Microbiol. 8, 2224 (2017).](https://www.frontiersin.org/articles/10.3389/fmicb.2017.02224/)
- Yerramsetty, K. Relative vs Absolute: Understanding Compositional Data with Simulations (Medium article)
  - [Part 1](https://towardsdatascience.com/relative-vs-absolute-understanding-compositional-data-with-simulations-fdc15e0c781e)
  - [Part 2](https://medium.com/@krishna.yerramsetty/relative-vs-absolute-how-to-do-compositional-data-analyses-part-2-f554eb9b26e)

#### 2016
- [Weiss, S. et al. Correlation detection strategies in microbial data sets vary widely in sensitivity and precision. ISME J. 10, 1669 (2016).](https://www.nature.com/articles/ismej2015235)

#### 2014
- [Buttigieg, P. L. & Ramette, A. A guide to statistical analysis in microbial ecology: A community-focused, living review of multivariate data analyses. FEMS Microbiol. Ecol. 90, 543–550 (2014).](https://boris.unibe.ch/91368/)
- [McMurdie, P. J. & Holmes, S. Waste Not, Want Not: Why Rarefying Microbiome Data Is Inadmissible. PLoS Comput. Biol. 10, (2014).](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003531)

#### 2013
- [McMurdie, P. J. & Holmes, S. Phyloseq: An R Package for Reproducible Interactive Analysis and Graphics of Microbiome Census Data. PLoS One 8, (2013).](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0061217)

